packages/vm/lib/index.ts

```diff
@@ -192,7 +192,7 @@ export default class VM extends AsyncEventEmitter {
    * reverted if an exception is raised. If it's `false`, it won't revert if the block's header is
    * invalid. If an error is thrown from an event handler, the state may or may not be reverted.
    *
-   * @param opts - Default values for options:
+   * @param {RunBlockOpts} opts - Default values for options:
```

Comment:
Ah, I actually *removed*  all these type notes on the MPT PR https://github.com/ethereumjs/merkle-patricia-tree/pull/125 😀  😛

For `TypeDoc` this is not necessary since it is read from the function signatures - see [docs](https://typedoc.org/guides/doccomments/) - and we should actually remove this from all code since it is redundant information and increases the chance to introduce some doc inconsistencies when this would be forgotten to be updated on signature changes (and it is just plainly unnecessary extra work as well).

So we should these type additions from all or selected libraries all together along some subsequent PRs. Will not make this a blocker on this round, since this is the only real addition and this needs to be addressed more systematically anyhow.

Reply:
Update: ah, 2-3 more additions, anyhow.